### ISSUE TITLE
Inline csslint rules affect other files

### ISSUE THREAD

**Issue by Krinkle** [gruntjs/grunt-contrib-csslint#26]
When overriding a csslint rule in a file, it also affects files checked after that one. I don't think that's intentional behaviour. Even more so when considering the order of the files is somewhat arbitrary (especially when using file patterns).

Example:

``` css
-- foo.css
.ooui-bulletlist {
    background-image: url(images/icons/bullet-x.svg);
}

.veui-block .ooui-bulletlist {
    background-image: url(images/icons/bullet-y.svg);
}

.mwui-block .ooui-bulletlist {
    background-image: url(images/icons/bullet-x.svg);
}

-- quux.css
.ooui-bulletlist {
    background-image: url(images/icons/bullet-x.png);
}

.veui-block .ooui-bulletlist {
    background-image: url(images/icons/bullet-y.png);
}

.mwui-block .ooui-bulletlist {
    background-image: url(images/icons/bullet-x.png);
}
```

This (rightfully) results in the following warnings:

> foo.css: Background image 'images/icons/bullet-x.svg' was used multiple times, first declared at line ..
> quux.css: Background image 'images/icons/bullet-x.png' was used multiple times, first declared at line ..

Then, if I add `/*csslint duplicate-background-images:false */` (e.g. if I would want to suppress this warning for some reason) in the first file and run `grunt csslint` again, both warnings disappear.

When moved from the top of `foo.css` to `quux.css`, the warning from foo.css showed up again, so it does seem like it is somehow being remembered throughout the task execution.
